### PR TITLE
Call intermittent checker

### PR DESCRIPTION
**Pull request by notriddle**
Part of servo/servo#12629

---
This change is [Reviewable](https://reviewable.io/reviews/servo/saltfs/453)

### CODE REVIEW

**Comment by aneeshusa** [comment]
You'll also need to update `buildbot/master/files/config/master.cfg` to add new Buildbot Builders so Buildbot knows to schedule + run the jobs.

File: buildbot/master/files/config/steps.yml [hunk 1]
```diff
@@ -27,6 +27,14 @@ mac-nightly:
   - ./mach package --release
   - ./etc/ci/upload_nightly.sh mac
 
+linux-rel-intermittent:
+  - ./mach build --release
+  - ./etc/ci/check_intermittents.sh
```
Comment (aneeshusa) [inline buildbot/master/files/config/steps.yml]: You can add `--log-raw intermittents.log` so that Buildbot will pick up the log file automatically.

**Comment by aneeshusa** [comment]
@asajeffrey I think you wanted the intermittent checker to run nightly instead of on each build, right?

**Comment by asajeffrey** [comment]
Indeed, we're running the intermittents 300 times each!

**Comment by aneeshusa** [comment]
r=me after squash and the `servo/servo` PR lands.

**Comment by notriddle** [comment]
Squished.

**Comment by aneeshusa** [comment]
servo/servo#12629 landed.

@bors-servo r+

**Comment by aneeshusa** [comment]
@bors-servo retry

- #467 
- #465

**Pull request merged**